app/lib/sms-games/controllers/SGCreateFromMobileController.js

```diff
@@ -65,110 +64,105 @@ SGCreateFromMobileController.prototype.processRequest = function(request, respon
     return false;
   }
 
-  // Query for an existing game creation config doc.
-  var queryConfig = configModel.findOne({alpha_mobile: request.body.phone});
-  var promiseConfig = queryConfig.exec();
-  promiseConfig.then(function(configDoc) {
-    // If no document found, then create one. This game creation config doc 
-    // should NOT be confused with the game doc, or the gameConfig. It's destroyed
-    // after the game is begun; it's used only for game creation. 
-    if (configDoc == null) {
-      // We don't ask for the first name yet, so just saving it as phone for now.
-      var doc = {
-        alpha_mobile: request.body.phone,
-        story_id: request.query.story_id,
-        story_type: request.query.story_type,
-        game_type: (request.query.game_type || '')
-      };
-
-      // User should have responded with the alpha name.
-      var message = request.body.args;
-      if (smsHelper.hasLetters(message)) {
-        doc.alpha_first_name = smsHelper.getLetters(message);
-        // Send next message asking for beta_mobile_1.
-        sendSMS(request.body.phone, gameConfig.mobile_create.ask_beta_0_oip);
-      }
-      else {
-        // If user responded with something else, ask for a valid player name. 
-        sendSMS(request.body.phone, gameConfig.mobile_create.invalid_alpha_first_name);
-      }
-      var createGame = configModel.create(doc);
-      createGame.then(function(doc) {
+  // If the request query params contain the 'alpha_first_name_query_response' param, 
+  // check whether or not we should create a new create game config. 
+  // (not to be confused with the game doc, or the gameConfig--it's destroyed after
+  // the game is begun; it's only used for game creation.)
+  if (request.query.alpha_first_name_query_response) {
+    var doc = {
+      alpha_mobile: request.body.phone,
+      story_id: request.query.story_id,
+      story_type: request.query.story_type,
+      game_type: (request.query.game_type || '')
+    };
+    var message = request.body.args;
+    // User should have responded with the alpha name. If so, create game create config doc.
+    if (smsHelper.hasLetters(message)) {
+      doc.alpha_first_name = smsHelper.getLetters(message);
+      var createGameConfig = configModel.create(doc);
```

Comment:
Potential `TODO` here is to also delete any existing create-game docs for this user number and game id.